cmake/linker_script/common/common-ram.cmake

```diff
@@ -121,7 +121,7 @@ endif()
 if(CONFIG_USB_DEVICE_STACK OR CONFIG_USB_DEVICE_STACK_NEXT)
   zephyr_iterable_section(NAME usb_cfg_data GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
   zephyr_iterable_section(NAME usbd_contex GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
-  zephyr_iterable_section(NAME usbd_class_node GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
+  zephyr_iterable_section(NAME usbd_class_iter GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
```

Comment:
Missing changes in `include/zephyr/linker/common-ram.ld`

Reply:
What exactly do you mean? `usbd_class_node` never was in `include/zephyr/linker/common-ram.ld`, the changes are in `subsys/usb/device_next/usbd_data.ld`.

Reply:
Was not aware it was in a separate file